### ISSUE TITLE
Variable changes and addition for VHFs

### ISSUE THREAD

**Issue by chukwuemekauzoho** [hzi-braunschweig/SORMAS-Project#1447]
@MartinWahnschaffeSymeda @danieltomaba @Chinedar ; Based on the recent development in the Lassa Fever TWG, a few variable changes have been made on the CIF forms for VHFs. There's a request for this to reflect and tally with the SORMAS platform as the CIF forms are sent to the states. Below are the questions added :

**Under Present Condition**
- Alive i. At home ii. On admission
- Dead i. Died at home ii. Died on arrival/at the hospital

**Under epidemiological info**
Instead of Area type (Area/urban) - replace with- 
-What type of house or shelter do you live in, select as appropriate?
i. Mud house
ii. Brick house
iii. Temporary shelter (Camps)
iv. Hostel
v. Other

**Under Hospitalization**
-Add ever tested positive for any VHF before...(Yes/No) -
i. Which VHF.... (Lassa, Ebola, etc)

**Under Epidemiological data*
-Do you or your family dry food materials in the open, e.g. rice, garri, fish, utensils, etc.
-Do you or your family leave containers of foodstuff uncovered?

**Comment by chukwuemekauzoho** [comment]
@tomabadaniel @MartinWahnschaffeSymeda @Chinedar @bernardsilenou the above update for the CIF did not reflect on the new release.

**Comment by MateStrysewskeSym** [comment]
@chukwuemekauzoho That's correct. We document everything that has been released in our release notes (https://github.com/hzi-braunschweig/SORMAS-Project/releases) and always close the issues that have been resolved.

**Comment by bernardsilenou** [comment]
Here are some suggestions that we can discuss in next sprint.
1.  Under outcome variable, if the outcome is "dead", then we can add a caption called  "Place of death" which will have categories: "Hospital, home, others, unknown"
2. If the outcome is "alive", then the heath facility variable of the case  already tell us if the person is at home or at HF, so e do not need to change this variable
3 Under the case tab, we can add:
    Was case ever tested positive for any VHF before...(Yes/No) -
    If yes, Which VHF.... (Free text)
For the other points, @chukwuemekauzoho will sent the current CIF for Lassa for use to have a look before identifying the changes to make.

**Comment by chukwuemekauzoho** [comment]
@bernardsilenou I have attached the Case investigation form below
Also noted adding pregnancy to export function as discussed
[Lassa Fever Case Investigation Form final Jan 2020 (1).docx](https://github.com/hzi-braunschweig/SORMAS-Project/files/4180471/Lassa.Fever.Case.Investigation.Form.final.Jan.2020.1.docx)

**Comment by bernardsilenou** [comment]
Danke, 
We will have a look at the from in the next sprint

**Comment by chukwuemekauzoho** [comment]
@bernardsilenou @chinedu, please can we add sample ID to the detailed export

**Comment by MateStrysewskeSym** [comment]
We discussed this today:
* The details fields for the present condition "dead" are already in the system, but only for EVD; we will enable this for other VHFs
* The detailed information about "alive" persons is already in there (Hospitalization and the Health Facility field under Case); we will add an information sidebar to the different case tabs in the future so that this is always visible no matter where the user currently is located at
* We will add the "Ever tested for any VHF before" item to epidemiological data instead of hospitalization

For all VHFs:

- [ ] Enable death details fields for all VHFs
- [ ] Add type of house or shelter
- [ ] Add "Ever tested for any VHF before" under epidemiological data, text field to put in the diseases
- [ ] Add the two new epidemiological data items

**Comment by bernardsilenou** [comment]
@chukwuemekauzoho 
In association to Matés' comment above, we have on our correctly building a case control module for users to be able to add any exposure or epidemiological data that they wish to collect in addition to the SORMAS default variables. We will ask the Lassa tam to be patient for a while untill the module is ready. 
With the  case control module, users will be able to collect these exposure data for controls also. Collecting them for cases only will not help in any epidemiological investigation since there is no denominator to compare with.
We will update the outcome variable as requested in the next release. Please if there is anything else, Let us have a call.

**Comment by MateStrysewskeSym** [comment]
@bernardsilenou If I understand you correctly, that's not true. Allowing users to add custom variables to any of the existing forms is something that is planned, but won't likely be implemented in the foreseeable future. I suggest that we should go ahead with the implementation of these requested variables if they're needed. Please discuss this internally again and get back to us.

**Comment by bernardsilenou** [comment]
@MateStrysewskeSym What you said is absolutely correct . This module will take some time in oder to be ready to use. 
Since SORMAS  is currently a disease surveillance system with data on cases only, collecting more exposure or epid data like housing type and food type will not be helpful because we do not have the same data for controls (non cases) in order to compare. Even if the researcher decides to do a case only study, then we need to collect the data at the time before the cases became cases for which to not have in SORMAS. Lets keep it simple and collect only data that can be used to direct immediate public health intervention for now and keep working on the case control module.
